deltachat-ffi/src/lib.rs

```diff
@@ -684,7 +684,11 @@ pub unsafe extern "C" fn dc_get_msg<'a>(
 
 #[no_mangle]
 pub unsafe extern "C" fn dc_may_be_valid_addr(addr: *mut libc::c_char) -> libc::c_int {
-    dc_contact::dc_may_be_valid_addr(addr)
+    if dc_contact::dc_may_be_valid_addr(addr) {
```

Comment:
@Hocuri @ralphtheninja this should be written as  
```rust
dc_contact::dc_may_be_valid_addr(addr) as libc::c_int
```

Reply:
Aye, I figured we could do something like this that was a bit prettier. I'll fix.